data/Mvishal123.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Vishal",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open-source | DSA | Blog | Freelance | Loading..",
+  "avatar": "https://twitter.com/Mvishalx.png",
```

Comment:
Please fix the broken avatar link. Twitter doesn't allow you to use pfp link in this way! Instead, it's better to use the default one. 🛠

```suggestion
  "avatar": "https://github.com/Mvishal123.png",
```